kani-driver/src/project.rs

```diff
@@ -100,6 +99,56 @@ impl Project {
                 && expected_path.as_ref().map_or(true, |goto_file| *goto_file == artifact.path)
         })
     }
+
+    /// Try to build a new project from the build result metadata.
+    ///
+    /// This method will parse the metadata in order to gather all artifacts generated by the
+    /// compiler.
+    fn try_new(
+        session: &KaniSession,
+        outdir: PathBuf,
+        metadata: Vec<KaniMetadata>,
+        cargo_metadata: Option<cargo_metadata::Metadata>,
+        failed_targets: Option<Vec<String>>,
+    ) -> Result<Self> {
+        // For each harness (test or proof) from each metadata, read the path for the goto
+        // SymTabGoto file. Use that path to find all the other artifacts.
+        let mut artifacts = vec![];
+        for crate_metadata in &metadata {
+            for harness_metadata in
+                crate_metadata.test_harnesses.iter().chain(crate_metadata.proof_harnesses.iter())
+            {
+                let symtab_out = Artifact::try_new(
+                    harness_metadata.goto_file.as_ref().expect("Expected a mode file"),
```

Comment:
What does "mode" in the expect message refer to? Should it be "model"?

Reply:
It's a typo. It should be model.